### ISSUE TITLE
[BUG] Warning: the imp module is deprecated in favour of importlib

### ISSUE THREAD

**Issue by johanburati** [Azure/WALinuxAgent#1473]
Getting a warning on Ubuntu 19.04 

**details:**
```
$ waagent --version
/usr/sbin/waagent:27: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
WALinuxAgent-2.2.32.2 running on ubuntu 19.04
Python: 3.7.2
Goal state agent: 2.2.37

$ cat /etc/os-release
NAME="Ubuntu"
VERSION="19.04 (Disco Dingo)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu Disco Dingo (development branch)"
VERSION_ID="19.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=disco
UBUNTU_CODENAME=disco
```

**Comment by vrdmr** [comment]
@roiyz-msft  - It is imported by the `waagent` file in Ubuntu's new VMs (which only have Py3 by default)

```
import os
import imp
import sys

if __name__ == '__main__' :
    import azurelinuxagent.agent as agent
    """
    Invoke main method of agent
    """
    agent.main()
```

Deprecation warning.
```
>>> import imp
__main__:1: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
>>> 
```

**Comment by amkarmak** [comment]
I am also getting the same error.. 

root@ubuntu19:~# waagent --version
/usr/sbin/waagent:27: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
WALinuxAgent-2.2.40 running on ubuntu 19.04
Python: 3.7.3
Goal state agent: 2.2.45
root@ubuntu19:~#

Anyone got any fix for this issue?

**Comment by narrieta** [comment]
@amkarmak - This should just be a warning, is the agent somehow failing? Thanks

**Comment by amkarmak** [comment]
Hi Norberto, no the agent is not failing. But this messages gets popped up.

**Comment by amkarmak** [comment]
Hi Johan, thank you for the response. Sure, I will do this and update.

**Comment by pgombar** [comment]
Fixed with #1738

**Comment by lewtwo** [comment]
Same Error with vm workstation player:
===================
$ cd /home/lewis/Downloads
$ sudo ./VMware-Player-15.5.6-16341506.x86_64.bundle
[sudo] password for lewis:        
Extracting VMware Installer...done.
/tmp/tmpbxds2o8v.vmis.env:4: DeprecationWarning: **the imp module is deprecated in favour of importlib**; see the module's documentation for alternative uses
  import imp
/tmp/vmis.iAlRZf/install/vmware-installer/vmis-launcher: error while loading shared libraries: libcrypto.so.1.0.2: cannot open shared object file: No such file or directory
====================
